test/integration/push_test.rb

```diff
@@ -27,6 +27,11 @@ class PushTest < ActionDispatch::IntegrationTest
     css = %(div.gem__users a[alt=#{@user.handle}])
 
     assert page.has_css?(css, count: 2)
+
+    assert_equal Digest::MD5.hexdigest(<<~INFO), Rubygem.find_by!(name: "sandworm").versions.sole.info_checksum
```

Comment:
@simi doesn't this cover the issue? before it was different here, since the info checksum reflected an empty info file